v2/datastream-to-spanner/terraform/samples/monitoring-alerting/modules/alerting/dataflow/dataflow.tf

```diff
@@ -0,0 +1,114 @@
+# Log-based metric for tracking Dataflow conversion errors separately
+resource "google_logging_metric" "dataflow_conversion_errors_metric" {
+  name = "dataflow_conversion_errors_metric"
+
+  # Filter to capture only conversion errors with severity of ERROR or higher for Dataflow jobs
+  filter = "resource.type=\"dataflow_job\" AND severity>=ERROR AND textPayload:\"conversion error\""
```

Comment:
@shreyakhajanchi Please validate with the recent updates to the metrics if the `filter` is still accurate.